### PR TITLE
Native CI builds

### PR DESCRIPTION
**Pull request by davidliu**
As a sanity test, I made some github actions to compile the GumTestApp (which includes compiling the main webrtc lib).

### CODE REVIEW

**Comment by davidliu** [comment]
Example of the actions working:

Android: https://github.com/davidliu/react-native-webrtc/actions/runs/3928901009/jobs/6717052995
iOS: https://github.com/davidliu/react-native-webrtc/actions/runs/3928901016

File: .github/workflows/android_ci.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+name: Android CI
+on:
+  push:
+    branches: [ main ]
```
Comment (saghul) [inline .github/workflows/android_ci.yml:4]: master

File: .github/workflows/ios_ci.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+name: iOS CI
+on:
+  push:
+    branches: [ main ]
```
Comment (saghul) [inline .github/workflows/ios_ci.yml:4]: ditto, master

**Review by saghul — COMMENTED** [review]
A couple of comments, LGTM otherwise. Thanks for doing this!

**Pull request merged**